frontend/src/components/WorksheetExpandedList.js

```diff
@@ -32,148 +34,13 @@ const WorksheetExpandedList = ({
-          >
-            <React.Fragment>
-              {/* Course Title, Code, and Skills/Areas */}
-              <div
-                style={{
-                  lineHeight: '30px',
-                  width: `${
-                    ROW_WIDTH -
-                    (width > PROF_CUT ? PROF_WIDTH : 0) -
-                    (width > MEET_CUT ? MEET_WIDTH : 0) -
-                    3 * RATE_WIDTH -
-                    BOOKMARK_WIDTH -
-                    PADDING
-                  }px`,
-                  paddingLeft: '15px',
-                }}
-              >
-                <strong>{'Course'}</strong>
-              </div>
-              {/* Course Professors */}
-              {width > PROF_CUT && (
-                <div
-                  style={{ lineHeight: '30px', width: `${PROF_WIDTH}px` }}
-                  className="pr-4"
-                >
-                  <strong>{'Professors'}</strong>
-                </div>
-              )}
-              {/* Course Meet Times and Location */}
-              {width > MEET_CUT && (
-                <div style={{ lineHeight: '30px', width: `${MEET_WIDTH}px` }}>
-                  <strong>{'Meets'}</strong>
-                </div>
-              )}
-              {/* Class Rating */}
-              <div
-                style={{ lineHeight: '30px', width: `${RATE_WIDTH}px` }}
-                className="d-flex"
-              >
-                <strong className="m-auto">{'Class'}</strong>
-              </div>
-              {/* Professor Rating */}
-              <div
-                style={{ lineHeight: '30px', width: `${RATE_WIDTH}px` }}
-                className="d-flex"
-              >
-                <strong className="m-auto">{'Prof'}</strong>
-              </div>
-              {/* Workload Rating */}
-              <div
-                style={{ lineHeight: '30px', width: `${RATE_WIDTH}px` }}
-                className="d-flex"
-              >
-                <strong className="m-auto">{'Work'}</strong>
-              </div>
-            </React.Fragment>
-            {/* Settings Dropdown to switch season and FB friend worksheet */}
-            <div
-              style={{
-                lineHeight: '30px',
-                width: `${BOOKMARK_WIDTH}px`,
-                paddingRight: '15px',
-              }}
-              className="d-flex"
-            >
-              <WorksheetSettingsDropdown
-                cur_season={cur_season}
-                season_codes={season_codes}
-                onSeasonChange={onSeasonChange}
-                setFbPerson={setFbPerson}
-                cur_person={fb_person}
-              />
-            </div>
-          </Row>
-        </div>
-        {/* Render HTML of items */}
-        <div className={styles.results_list_container}>{items}</div>
-      </Container>
-    </div>
+    <SearchResults
+      data={courses}
+      showModal={showModal}
+      expanded={cur_expand === 'list'}
+      isLoggedIn={false}
```

Comment:
wat

Reply:
oops the isLoggedIn should always be true cuz u can only access the worksheet if logged in. Other than that, it's just required props for SearchResults